VERSION "0.0.1"


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: VCU SB1 SB2 COOLING_CTRL INVERTER SIM


BO_ 13 SIM_pps_sensors: 4 SIM
 SG_ pps_inv : 13|13@1+ (1,0) [0|5000] "mV"  SB1
 SG_ pps : 0|13@1+ (1,0) [0|5000] "mV"  SB1

BO_ 29 SIM_cooling_sensors: 2 SIM
 SG_ thermocouple : 0|13@1+ (1,0) [0|5000] "mV"  SB2

BO_ 19 COOLING_CTRL_fan_ctrl: 1 COOLING_CTRL
 SG_ fan_pwm_duty : 0|7@1+ (1,0) [0|100] "%"  SIM




CM_ BO_ 13 "pps readings";
CM_ SG_ 13 pps_inv "inverting PPS voltage";
CM_ SG_ 13 pps "non-inverting PPS voltage";
CM_ BO_ 29 "thermocouple voltage";
CM_ SG_ 29 thermocouple "Voltage on the thermocouple end";
CM_ BO_ 19 "fan pwm control";
CM_ SG_ 19 fan_pwm_duty "fan PWM duty cycle";
BA_DEF_ BO_  "GenMsgCycleTime" INT 0 65535;
BA_DEF_DEF_  "GenMsgCycleTime" 0;
BA_ "GenMsgCycleTime" BO_ 19 100;




